site isn't building

something about a missing cobertura report:

```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Jaxen 2.0.2-SNAPSHOT:
[INFO] 
[INFO] Jaxen .............................................. SUCCESS [ 35.962 s]
[INFO] jaxen .............................................. FAILURE [ 20.338 s]
[INFO] jaxen-integration .................................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  56.675 s
[INFO] Finished at: 2026-04-18T09:31:56-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.12.1:site (default-cli) on project jaxen: failed to get report for org.codehaus.mojo:cobertura-maven-plugin: Failed to execute goal com.github.siom79.japicmp:japicmp-maven-plugin:0.23.1:cmp (check-jaxen-api-compatibility) on project jaxen: There is at least one modified class: org.jaxen.BaseXPath -> [Help 1]
[ERROR]
```

--- a/pom.xml
+++ b/pom.xml
@@ -295,11 +295,6 @@
       </plugin>
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
-        <artifactId>cobertura-maven-plugin</artifactId>
-        <version>2.7</version>
-      </plugin>
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
         <artifactId>findbugs-maven-plugin</artifactId>
         <version>3.0.5</version>
       </plugin>
@@ -431,11 +426,6 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-report-plugin</artifactId>
         <version>3.5.1</version>
-      </plugin>
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>cobertura-maven-plugin</artifactId>
-        <version>2.7</version>
       </plugin>
       <plugin>
         <groupId>org.codehaus.mojo</groupId>

